pathd/pceplib.lock

```diff
@@ -0,0 +1 @@
+346ff342e7fac30e70280123ea9ebafd3c2bb686
```

Comment:
This is probably not clear enough, but this is the commit hash of pceplib that pathd was tested with. I though this was somewhere in the doc...

Reply:
making dependencies with 3rd parties libraries should be done with appropriate versioning. pcelib should be produced accordingly.

Reply:
Sound good, is there some doc or guide on how this appropriate versioning should be done ?

Reply:
well, I thought about it, you would need a binary repository where you could store the pce library package(maybe with help from Martin).
about packaging pcelib in package, I can give you some link where I scripted the packaging of some libraries I needed.
Perhaps it can help for you (or for Brady, I guess).
https://github.com/6WIND/zrpcd/blob/master/pkgsrc/packaging.sh

Reply:
Is really a package/binary version of pceplib required for this PR to get merge ? The pcep module is optional, and experimental, I was assuming that if building from source is required for now, this wouldn't be an issue.